### PR TITLE
Add principal approval email for submitted teacher applications

### PR DESCRIPTION
**Pull request by aoby**
And fixed several small bugs in the teacher application.

### CODE REVIEW

**Comment by aoby** [comment]
[Emails triggered after submitted teacher application](https://trello.com/c/xkS4PewF/243-emails-triggered-after-submitted-teacher-application)

**Comment by aoby** [comment]
[Emails triggered after submitted teacher application](https://trello.com/c/xkS4PewF/243-emails-triggered-after-submitted-teacher-application)

File: dashboard/app/views/pd/application/teacher1819_application_mailer/principal_approval.html.haml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+%p
+  Dear
+  = @application.principal_title_and_last_name + ','
```
Comment (clareconstantine) [inline dashboard/app/views/pd/application/teacher1819_application_mailer/principal_approval.html.haml]: What if they don't have a title? (not sure if that's possible with the form setup) Do we need a fallback to full name?

Comment (aoby) [reply dashboard/app/views/pd/application/teacher1819_application_mailer/principal_approval.html.haml]: Good point! In the original spec, title was required but we nixed that. @tanyaparker How should we address the email when there is no title? Full name? First name?

Comment (tanyaparker) [reply dashboard/app/views/pd/application/teacher1819_application_mailer/principal_approval.html.haml]: Let's do full name.

**Pull request merged**